Common/dump_all_test_ids.sh

```diff
@@ -34,7 +34,7 @@ echo ">> ***********************************"
 
 echo ""
 
-for setup in {1..11}
+for setup in {2..11}
```

Comment:
We will have to revisit the setups numbering.

Reply:
Absolutely